test/webrick/test_httprequest.rb

```diff
@@ -77,7 +77,7 @@ def test_request_uri_too_large
     _end_of_message_
     req = WEBrick::HTTPRequest.new(WEBrick::Config::HTTP)
     assert_raise(WEBrick::HTTPStatus::RequestURITooLarge){
-      req.parse(StringIO.new(msg.gsub(/^ {6}/, "")))
+      req.parse(StringIO.new(msg.gsub(/^ {6}/, "").gsub("\n", "\r\n")))
```

Comment:
There is a lot of duplicated setup. Would it make sense to extract it into a helper?

Reply:
Sure.  If someone wants to work on cleaning up the specs in a later PR, that would be great.